dev/ColorPicker/readme.md

```diff
@@ -0,0 +1,141 @@
+# ColorPicker Architecture
+
+This document is primarily intended for developers and contributors to WinUI. For additional information see:
```

Comment:
nit: For information on how to use the ColorPicker see:

If you are already in the source code of WinUI and see the readme of ColorPicker, either you need this information and the API/design docs aren't that helpful anymore to you or you are "lost" and actually want to use the control, not modify it, that's at least how I would see it.

Reply:
Being overly specific here isn't needed. (1) This section could include any related links other than those listed (2) As a developer you most likely already know the current links therefore it's there to save time only.

Reply:
Fair point.